opennlp-tools/src/test/java/opennlp/tools/util/featuregen/GeneratorFactoryTest.java

```diff
@@ -124,4 +125,16 @@ public void testArtifactToSerializerMappingExtraction() throws IOException {
 
     Assert.assertTrue(mapping.get("test.resource") instanceof WordClusterDictionarySerializer);
   }
+
+  @Test
+  public void testDictionaryArtifactToSerializerMappingExtraction() throws IOException {
+
+    InputStream descIn = getClass().getResourceAsStream(
```

Comment:
should probably also add it to "testArtifactToSerializerMappingExtraction" too.. I kinda stole my test from that one.